pvc_to_pvs_image_import/_topics/preparing_powervc_node.md

```diff
@@ -0,0 +1,222 @@
+### General Assumptions:
+
+1. You have the latest installation of the {{ site.data.product.title_short }} base (>= [Morphy](https://github.com/{{ site.data.product.title_short }}/manageiq/milestone/4)) on your server or workstation.
+
+
+
+
+2. You have `ansible-runner` installed in the system that hosts your {{ site.data.product.title_short }} instance.  
+For installation instructions please refer to this [guide](https://ansible-runner.readthedocs.io/en/stable/install.html). 
+
+
+3. Your {{ site.data.product.title_short }} base is located under `<MIQ_PATH>` on your {{ site.data.product.title_short }} hosting site.
+ 
+
+4. You have chosen a PowerVC control server for usage with the import workflow and have administrative access via it's command line.
+
+
+
+### Preparing IBM PowerVC Server (required once):
```

Comment:
Probably is also a good idea to ensure the PowerVC system is at some minimum software level--suggest putting min version at the version against which you're testing and we go from there and newer.  Earlier versions would work, but this is to limit surface area to tested scenarios.